src/pages/documentation/release-notes/index.md

```diff
@@ -4,6 +4,26 @@ description: Release notes and change logs for the Adobe Experience Platform Mob
 
 # Release notes
 
+## Aug 7, 2023
+
+### Roku SDK 1.0.0
+
+This is the first stable release of the Roku SDK, which exposes APIs to help developers to integrate the Adobe Experience Platform solutions with the Roku channel.
+
+Features:
+
+* **Adobe Experience Edge Network Integration:** Seamlessly send data to the Adobe Experience Edge Network. Optionally include an Experience Edge response callback using the API for greater control over data interactions.
```

Comment:
Adobe Experience Platform Edge Network integration

Seamlessly send data to the Edge Network, and get greater control over data interactions by optionally registering a callback for Edge response handles.